partner_communication_switzerland/data/auto_reminder_archive_contact.xml

```diff
@@ -0,0 +1,30 @@
+<odoo>
+    <data noupdate="1">
+        <record id="auto_reminder_archive_contact_template" model="mail.template">
+            <field name="name">Archive contact reminder mail</field>
+            <field
+        name="model_id"
+        ref="partner_communication.model_partner_communication_job"
+      />
+            <field
+        name="email_from"
+      >&lt;compassion@compassion.ch&gt;</field>
+            <field name="reply_to">sds@compassion.ch</field>
+            <field name="use_default_to" eval="True" />
+            <field
+        name="subject"
+      >Reminder: Invalid partners</field>
+            <field name="body_html" type="html">
+                <div>
+                    <b>The following partners meet the criteria and are considered invalid : </b>
+                    % set partner_info = object.partner_id.get_archive_contact()
+                    <span style="font-family: 'miller';">
+                    % for partner in partner_info:
+                        <p>ID ${partner.id} - ${partner.name}</p>
```

Comment:
I think that the partner reference is more meaningful to SDS than the database ID. You could also include a direct link to the contact:
```suggestion
                        <p><a href="/web#id=${partner.id}&model=res.partner&view_type=form&menu_id=1744">Ref ${partner.ref} - ${partner.name}</a></p>
```